LibraBFT/Impl/Properties/VotesOnceDirect.agda

```diff
@@ -78,11 +78,11 @@ module LibraBFT.Impl.Properties.VotesOnceDirect where
 
   votesOnce₁ : VO.ImplObligation₁
   votesOnce₁ {pid' = pid'} r (step-msg {_ , P m} _ psI) {v' = v'} {m' = m'}
-             pkH v⊂m (here refl) sv ¬msb vspkv v'⊂m' m'∈pool sv' ep≡ r≡
```

Comment:
These changes should be reverted (see other comment about `ValidSenderForPK⇒ep≡`).  Also, maybe change `ep≡` to `eid≡` to be more clear.

Reply:
DONE